MovieEnthusiast.Api.Tests/Controllers/MovieControllerTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using MovieEnthusiast.Api.Controllers;
+using MovieEnthusiast.Application.Common.Models;
+using MovieEnthusiast.Application.Queries;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace MovieEnthusiast.Api.Tests.Controllers;
+
+public class MovieControllerTests
+{
+    private IMediator _mediatorMock;
+    private MovieController _sut;
+    private Fixture _fixture;
+
+    [SetUp]
+    public void SetUp()
+    {
+        _mediatorMock = Substitute.For<IMediator>();
+        _sut = new MovieController(_mediatorMock);
+        _fixture = new Fixture();
+    }
+
+    [Test]
+    public async Task Valid_GetMovies_ReturnsOkResult()
```

Comment:
I don't see the value of this test. To me, Api.Tests would most likely qualify for integration testing (something like step 3 in https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-8.0#customize-webapplicationfactory) where you would interact and assert as a client of the API